CHANGELOG

```diff
@@ -1,26 +1,20 @@
-v3.17 ([MMM’YY])
+v3.17 (Month’20)
   * Use shared noscript partial
   * Upgraded gems: puma, sass-rails
   * Bugs fixed:
-    - [bug fixed #1]
-    - [bug fixed ...]
-    - Bug tracker items: #N, #M, #O, ...
+       - Long project names interfering with search bar expansion.
   * New integrations:
-    - [new integration #1]
-    - [new integration ...]
+       - 
   * Integration enhancements:
-    - Nexpose: better cipher wrapping coverage
-    - Nikto: support new nested <niktoscan> format
+       - 
   * Reporting enhancements:
```

Comment:
L7
L9
L12, L13

should not disappear.

Also worth looking into what's going on on L21-23

Reply:
✅